vmdb/app/models/mixins/container_provider_mixin.rb

```diff
@@ -6,11 +6,20 @@ module ContainerProviderMixin
     has_many :container_groups, :foreign_key => :ems_id, :dependent => :destroy
     has_many :container_services, :foreign_key => :ems_id, :dependent => :destroy
     has_many :container_replicators, :foreign_key => :ems_id, :dependent => :destroy
+    virtual_column :containers, :type => :container, :uses => :container_groups
```

Comment:
@zakiva , @simon3z , why did you choose to use virtual_column and not the "has_many containers :through..." approach? (http://guides.rubyonrails.org/association_basics.html#the-has-many-through-association)